src/SmokeTest/SmokeTest.csproj

```diff
@@ -13,4 +13,8 @@
     <Content Include="..\Particular.PlatformSample\platform\**\*" CopyToOutputDirectory="PreserveNewest" Link="platform\%(RecursiveDir)%(Filename)%(Extension)" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
To stick with the (admittedly undocumented) standard that's been established for our project files, This ItemGroup should come between the ProjectReference and Content ItemGroups.

Reply:
👍 I'll amend the commit